### PR TITLE
ssl keyapir automated.

### PR DESCRIPTION
**Pull request by alin2k**
osc.py getCertificates,
MC.robot has testcases 3.1-3.6 to upload 3 good keypairs and getCertificates check Issue name changed then roboot OSC and automation do OSC tests,
AL-240-156.robot defines three ssl-patch to upload 3 files.

### CODE REVIEW

File: lib/osc.py [hunk 1]
```diff
@@ -285,17 +285,13 @@ def _wait_for_job(self, data, headers="JSON"):
     # Get the ISC version.
     # Return a string with the ISC version
     def getISCVersion(self):
-        try:
-            self._output.log_info("inside getISCVersion")
-            action = 'ISC version query'
-            url = '/api/server/v1/serverManagement/status'
-            body = ''
+        action = 'ISC version query'
+        url = '/api/server/v1/serverManagement/status'
```
Comment (ychoma) [inline lib/osc.py]: Please don't change the getISCVersion method
It will revert my changes and make automation fail.

File: lib/osc.py [hunk 2]
```diff
@@ -477,14 +473,14 @@ def createOrUpdateOStackVC(self, vc, update_vcid=None, useLegacyProvAttrs=False)
             data = self._isc_connection(method=method, url=url, body=body, action=action, headers=headers)
             if data is None:
                 self._output.log_error("Exit createOStackVC -- Failed to create new VC")
-                return(None)
+                ##return(None)
```
Comment (ychoma) [inline lib/osc.py]: remove the ##

File: lib/osc.py [hunk 3]
```diff
@@ -477,14 +473,14 @@ def createOrUpdateOStackVC(self, vc, update_vcid=None, useLegacyProvAttrs=False)
...
             else:
                 vcid = data['id']
                 self._output.log_debug("Exit createOStackVC -- Created VC Id: \"%s\"" %(vcid))
-                return(vcid)
+                #return(vcid)
```
Comment (ychoma) [inline lib/osc.py]: remove the #

File: lib/osc.py [hunk 4]
```diff
@@ -477,14 +473,14 @@ def createOrUpdateOStackVC(self, vc, update_vcid=None, useLegacyProvAttrs=False)
...
             pass
 
-        # VC job is not spawn for create or update - there is nothing to check
+        return self._wait_for_job(data)
```
Comment (ychoma) [inline lib/osc.py]: change to "#VC job is not spawn for create or update - there is nothing to check"  without the quotes

File: lib/osc.py [hunk 5]
```diff
@@ -2464,7 +2462,7 @@ def uploadFileToOvf(self, rstUrl=None, method="POST", filePath=None, srcFd=None,
             filePath = urlparse(rstUrl).path
         pass
         fileName = os.path.basename(filePath)
-
+        ## base_path, fileName = os.path.split(filePath)
```
Comment (ychoma) [inline lib/osc.py]: remove this line

File: lib/osc.py [hunk 6]
```diff
@@ -2882,13 +2919,14 @@ def uploadSslKeypairImage(self, imgPath=None):
...
         if imgPath:
+            imgName = os.path.basename(imgPath)
             if not os.path.exists(imgPath):
                 raise Exception("uploadSslKeypairImage -- Error: Img File \"%s\" Not Found" %(imgPath))
             pass
             if not len(imgPath):
                 raise Exception("uploadSslKeypairImage -- Error: Img File \"%s\" is empty" %(imgPath))
             pass
-
+            #rstUrl = (ssl_upload_url + r"/" + imgName)
```
Comment (ychoma) [inline lib/osc.py]: remove this line

File: testbeds/AL-240-156.robot [hunk 7]
```diff
@@ -0,0 +1,134 @@
+
```
Comment (ychoma) [inline testbeds/AL-240-156.robot]: Remove this file we don't want to add it to GitHub

Instead just change Testbed.robot file to have the new line(s)

Comment (alin2k) [reply testbeds/AL-240-156.robot]: All changes done by rebase, edit, add, commit, fetch.

File: lib/osc.py (hunk 6 above)
Comment (alin2k) [reply lib/osc.py]: All changes done by rebase, edit, add, commit, fetch.

File: lib/osc.py (hunk 5 above)
Comment (alin2k) [reply lib/osc.py]: All changes done by rebase, edit, add, commit, fetch.

File: lib/osc.py (hunk 4 above)
Comment (alin2k) [reply lib/osc.py]: All changes done by rebase, edit, add, commit, fetch.

File: lib/osc.py (hunk 3 above)
Comment (alin2k) [reply lib/osc.py]: All changes done by rebase, edit, add, commit, fetch.

File: lib/osc.py (hunk 2 above)
Comment (alin2k) [reply lib/osc.py]: All changes done by rebase, edit, add, commit, fetch.

File: lib/osc.py (hunk 1 above)
Comment (alin2k) [reply lib/osc.py]: All changes done by rebase, edit, add, commit, fetch.

**Pull request merged**